db/migrate/20151111030421_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.text :body
+      t.references :post, index: true, foreign_key: true
```

Comment:
When you use `.references`, you actually don't need to do `index` and `foreign_key`. These lines are equivalent:

```rb
t.references :post
t.post_id index: true, foreign_key: true
```